scripts/start-finalExec

```diff
@@ -221,7 +221,11 @@ function copyFilesForCurseForge() {
   cp -f /data/eula.txt "${FTB_DIR}/"
 }
 
-if versionLessThan 1.7; then
+if versionLessThan 'b1.8'; then
```

Comment:
Now I'm confused by Mojang's version scheme. I thought beta was  before release 1.0. So I was expecting the 1.7 comparison to remain and you would add a new comparisons for < 1.3 which implies beta releases. However, the version comparison logic doesn't know about "b" prefix...unless you're enhancing that separately.

Reply:
I thought your version comparison uses [launchermeta](https://launchermeta.mojang.com/mc/game/version_manifest_v2.json) which has the Versions including Alpha and Beta in there and is/can be ordered.

I really did not look that far into that script. If you want to, I can but that will take some time.